doc/releases/release-notes-3.5.rst

```diff
@@ -650,6 +650,19 @@ Nanopb
 
   * Added helper cmake function ``zephyr_nanopb_sources`` to simplify ``.proto`` file inclusion.
 
+LVGL
+****
+
+  * Changed project status to maintained.
+
+  * Library has been updated to release v8.3.7.
+
+  * Added ``zephyr,lvgl-{pointer,button,encoder}-input`` pseudo device bindings.
+    :kconfig:option:`CONFIG_LV_Z_KSCAN_POINTER` is still supported but touch controllers
+    need a ``zephyr,kscan-input`` child node to emit input events.
+
+  * LVGL shell allows for monkey testing and inspecting memory usage.
```

Comment:
you could mention the Kconfig option here `:kconfig:option:`CONFIG_LV_USE_MONKEY``